src/cockpit/channels/dbus.py

```diff
@@ -409,9 +409,13 @@ async def handler(message):
                 name, props, invalids = message.get_body()
                 logger.debug('NOTIFY: %s %s %s %s', path, name, props, invalids)
                 for inv in invalids:
-                    reply, = await self.bus.call_method_async(self.name, path,
-                                                              'org.freedesktop.DBus.Properties', 'Get',
-                                                              'ss', name, inv)
+                    try:
+                        reply, = await self.bus.call_method_async(self.name, path,
+                                                                  'org.freedesktop.DBus.Properties', 'Get',
+                                                                  'ss', name, inv)
+                    except BusError as exc:
+                        logger.debug('failed to fetch property %s.%s on %s %s: %s',
+                                     name, inv, self.name, path, str(exc))
```

Comment:
This is missing at least a `continue`, as otherwise it would fall through the `props[inv] = reply` below. But presumably it should also something like `del props[inv]` in the error case?

Reply:
Ah, right!  I had the continue but I accidentally erased it when adding the logging.

From what I understand, `props` has just been unpacked from the message a few lines up.  That's the list of the properties that was just reported as changed, with their values.  It's theoretically possible that a property was reported in both places, I guess, but rather unlikely...

Reply:
OK, and even in that unlikely case, if it's part of the reported props, it's presumably alright.